src/index.js

```diff
@@ -0,0 +1,121 @@
+import './style.css';
+import Task from './modules/task.js';
+
+const tasks = document.querySelector('.task-items');
+
+const refresh = document.querySelector('#refersh');
+
+const clearAll = document.querySelector('#clear');
+
+const addNewTask = document.querySelector('#new-item');
+const enter = document.querySelector('#enter');
+const enterKey = document.querySelector('#new-item');
+
+let tasksList = JSON.parse(localStorage.getItem('tasks')) || [];
+
+// display tasks function
+const displayTask = () => {
+  tasks.innerHTML = tasksList.map((task) => `
+      <div id="${task.index}" class="task">
+        <div>
+            <input id="${task.index}" class="checkbox" type="checkbox" name="checkbox" ${!task.completed ? '' : 'checked'} />
+            <input id="task" type='text' class=" ${!task.completed ? '' : 'checked'} " value="${task.description}" />
+        </div>
+        <i id="ellips-btn" class="fa-solid fa-ellipsis-vertical ellips hidden"></i>
+        <i id="trash" class="fa-solid fa-trash trash"></i>
+    </div>
+      `).join('');
+};
```

Comment:
- [ ] According to the project guidelines, you should have an array of objects with each object having the following properties: description `(string)`, completed `(boolean)`, index `(number)`. Kindly create this.

- [ ] Your function should dynamically render the created array on the page.

  ![tours](https://user-images.githubusercontent.com/36542986/166928372-bb5f6858-2f41-43c5-b378-1dedca38bbcf.jpg)